userbenchmark/optim/run.py

```diff
@@ -368,11 +368,14 @@ def generate_random_gradients(parameters):
 def optimizer_step(optimizer):
     optimizer.step()
 
-def pt2_optimizer_step(optimizer):
+def compile_step(optimizer):
     @torchdynamo.optimize('inductor')
     def f():
         optimizer.step()
```

Comment:
This is what we want to measure for compile time, yes?

Reply:
In the framework, we use this code (https://github.com/pytorch/benchmark/blob/main/torchbenchmark/util/model.py#L356) to get compilation time and number of graph breaks.